app/policies/badge_policy.rb

```diff
@@ -18,6 +18,23 @@ def create?
   end
 
   def update?
-    person.managed_teams.include?(record.team) || user.admin
+    case record.permission
```

Comment:
For some reason whenever i try to update, with an `admin` account, it wouldn't let me